### DISCUSSION TITLE
CVE-2022-24348

### DISCUSSION THREAD

**Discussion by ggolub** [argoproj/argo-cd#8410] [Q&A] [answered]
https://github.com/argoproj/argo-cd/security/advisories/GHSA-63qx-x74g-jcr7 indicates that patches were released for 
- v2.3.0
- v2.2.4
- v2.1.9

What does this mean for 2.2.5, which is the version deployed by the v3.33.5 helm chart?  Are versions between 2.2.4 and 2.3.0 patched or are they still vulnerable?

**Comment by crenshaw-dev** [comment, marked as answer]
2.2.5 is patched, and subsequent 2.2.x versions will have the patch. All versions of 2.3.x will have the patch.
